### PR TITLE
feat(test)test: pull request

### PR DESCRIPTION
**Pull request by milediseifeddine**

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -24,7 +24,7 @@ pub enum Lit {
     N(Id)
 }
 
-impl Lit {
+impl Lit { //seif changes
```
Comment (milediseifeddine) [inline src/lib.rs]: test  test

File: src/lib.rs [hunk 2]
```diff
@@ -24,7 +24,7 @@ pub enum Lit {
     N(Id)
 }
```
Comment (michaelbeaumont) [inline src/lib.rs]: test test pleasechange this

File: src_other/lib.rs [hunk 3]
```diff
@@ -4,6 +4,7 @@ extern crate env_logger;
 extern crate vec_map;
 extern crate bit_set;
 
+// other comment
```
Comment (michaelbeaumont) [inline src_other/lib.rs]: bad comment!

File: src_other/lib.rs [hunk 4]
```diff
@@ -4,6 +4,7 @@ extern crate env_logger;
...
 use vec_map::VecMap;
 
 use Lit::{P, N};
```
Comment (michaelbeaumont) [inline src_other/lib.rs]: oother comment

**Pull request closed without merging**